lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -125,6 +125,7 @@ SessionState::SessionState(
       pdp_end_time_(marshaled.pdp_end_time),
       // 5G session version handlimg
       current_version_(marshaled.current_version),
+      throttle_count_(0),
```

Comment:
Can we have a comment here to indicate what this value is and how it is used?

Reply:
Not here, but in `SessionState.h`

Reply:
SMF-UPF version missmatch, max sending throttle count

Reply:
Can we have a comment to indicate that in SessionState.h please.

Reply:
Updated the same.